hockeysdk/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,8 @@
 
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" android:maxSdkVersion="18" />
```

Comment:
Hey, @laalto!
Thx for getting in touch and alerting us to this! I'm adding this to the release notes and to our documentation.

Benny